src/Template/Requests/view.ctp

```diff
@@ -16,6 +16,17 @@ use Cake\Core\Configure;
 </div>
 
 <ul id="toolbar" class="toolbar">
+     <li id="panel-button-left" class="panel hidden">
+        <span class="panel-button">
+            &lt;
+        </span>
+    </li>
+    <li id="panel-button-right" class="panel hidden">
+        <span class="panel-button">
+            &gt;
```

Comment:
What about the [fancy unicode left and right chevrons](http://www.fileformat.info/info/unicode/char/203a/index.htm)?